ui/src/experiments/list/search/SearchExperimentsPanel.scss

```diff
@@ -1,3 +1,6 @@
-.euiFlyoutFooter {
-  box-shadow: inset -5px 0 9px -9px rgba(0, 0, 0, 0.4);
+// Required for setting a specific flyout width that is smaller than the default EUI flyout width (medium); overwrites
+// the min-width that corresponds to the default EUI flyout width
+.euiFlyout--medium {
+  width: 313px;
+  min-width: unset;
```

Comment:
Moved the custom flyout widths here.

Reply:
Instead of applying this to all `euiFlyout--medium`, should we create a special class name `euiFlyout--searchPanel` and associate this to the flyout component and the styling?

Reply:
Fixed this here: https://github.com/caraml-dev/xp/blob/0a6fa979d645a743abc4c82f9293f6242a36ce52/ui/src/experiments/list/search/SearchExperimentsPanel.scss